drivers/gnss/Kconfig.quectel_lx6

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2025 Nick Ward <nix.ward@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config GNSS_QUECTEL_LX6
+	bool "Quectel LX6 GNSS modem driver"
+	default y
+	depends on GNSS
+	depends on DT_HAS_QUECTEL_L96_ENABLED
+	depends on GNSS_REFERENCE_FRAME_WGS84
+	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_QUECTEL_L96),i2c)
+	select SERIAL if $(dt_compat_on_bus,$(DT_COMPAT_QUECTEL_L96),uart)
+	select MODEM_MODULES
+	select MODEM_BACKEND_QUECTEL_I2C if $(dt_compat_on_bus,$(DT_COMPAT_QUECTEL_L96),i2c)
+	select MODEM_BACKEND_UART if $(dt_compat_on_bus,$(DT_COMPAT_QUECTEL_L96),uart)
+	select MODEM_CHAT
+	select GNSS_PARSE
+	select GNSS_NMEA0183
+	select GNSS_NMEA0183_MATCH
+	help
+	  Enable Quectel LX6 series GNSS modem driver.
+
+if GNSS_QUECTEL_LX6
+
+config GNSS_QUECTEL_LX6_RX_BUF_SIZE
+	int "Size of backend receive buffer"
+	default 256
+
+config GNSS_QUECTEL_LX6_TX_BUF_SIZE
+	int "Size of backend transmit buffer"
+	default 64
+
+config GNSS_I2C_POLL_MS
```

Comment:
This needs to be namespaced `GNSS_QUECTEL_LX6_` as well, it may become common in the future, at which time it should be moved to the drivers/gnss/Kconfig :)

Reply:
Done